src/routers/Gateway.sol

```diff
@@ -204,7 +211,24 @@ contract ConnectorGateway {
             (uint64 poolId, bytes16 trancheId, address destinationAddress, uint128 amount) =
                 ConnectorMessages.parseTransferTrancheTokens20(_msg);
             connector.handleTransferTrancheTokens(poolId, trancheId, destinationAddress, amount);
-        } else {
+        }  else if (ConnectorMessages.isCollectRedeem(_msg)) {
+            (uint64 poolId, bytes16 trancheId, address destinationAddress, uint128 currency, uint128 currencyPayout, uint128 tokensRedeemed, uint128 remainingRedeemOrder) =
+                ConnectorMessages.parseCollectRedeem(_msg);
+            connector.handleCollectRedeem(poolId, trancheId, destinationAddress, currency, currencyPayout, tokensRedeemed, remainingRedeemOrder);
+        } else if (ConnectorMessages.isCollectInvest(_msg)) {
```

Comment:
`CollectRedeem` comes before `CollectRedeem`, could we sort these statements accordingly to keep that order consistent across the repo? 🙏